lib/resources/dstu2/document_reference_create.yaml

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
General question to all: the cardinality shown here is directly from the HL7 website. Should these instead reflect our profiled cardinality? IE: 1..1 or 1..* for required?